packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -381,54 +335,6 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
 
     // internal code
 
-    /**
-     * @dev Initializes an account,
-     * stores it's public key, secret and counter,
-     * then emits {AccountInitialized} and {AccountSecretUpdated} events.
-     * @param account the address of the account
-     * @param uncompressedPubKey the public key of the account
-     * @param secret account's secret
-     */
-    function _initializeAccount(
-        address account,
-        bytes memory uncompressedPubKey,
-        bytes32 secret
-    ) internal {
-        Account storage accountData = accounts[account];
-
-        require(account != address(0), "account must not be empty");
-        require(ECDSA.uncompressedPubKeyToAddress(uncompressedPubKey) == account, "public key does not match account");
-        require(accountData.counter == 0, "account already initialized");
-        require(secret != bytes32(0), "secret must not be empty");
-
-        accountData.secret = secret;
-        accountData.counter = 1;
-
-        emit AccountInitialized(account, uncompressedPubKey, secret);
```

Comment:
We can't remove this completely because we need to map from addresses to public keys

Reply:
separate discussion.